conf/app.routes

```diff
@@ -45,7 +45,8 @@ POST       /client-money-protection-scheme                      controllers.Clie
 GET        /change-client-money-protection-scheme               controllers.ClientMoneyProtectionSchemeController.onPageLoad(mode: Mode = CheckMode)
 POST       /change-client-money-protection-scheme               controllers.ClientMoneyProtectionSchemeController.onSubmit(mode: Mode = CheckMode)
 
-GET        /dateOfChange                                        controllers.DateOfChangeController.onPageLoad(mode: Mode = NormalMode)
-POST       /dateOfChange                                        controllers.DateOfChangeController.onSubmit(mode: Mode = NormalMode)
-GET        /changeDateOfChange                                  controllers.DateOfChangeController.onPageLoad(mode: Mode = CheckMode)
-POST       /changeDateOfChange                                  controllers.DateOfChangeController.onSubmit(mode: Mode = CheckMode)
+GET        /date-of-change                                        controllers.DateOfChangeController.onPageLoad(mode: Mode = NormalMode)
```

Comment:
being really picky here but the indentation is not aligned with the rest of the file